docs/learning/api-support-policies.md

```diff
@@ -0,0 +1,91 @@
+    - Exception: changing the return type from `void` to something else.
+- Enums and tagged unions
+  - Removing values from an enum or types from a union.
+  - Adding new values to an enum or types to a tagged union, unless it is unlikely that existing code tests exhaustively for all possible values.
+
+The iTwin.js team strives to avoid breaking changes whenever possible. We also endeavor to design our APIs with flexibility to evolve without the need for breaking changes - for example, by defining functions to receive their parameters as a single object type instead of multiple individual arguments.
+
+## API categories
+
+iTwin.js uses [API Extractor](https://api-extractor.com/) to help manage its APIs. Each API exported by a package is marked with one of four ["release tags"](https://api-extractor.com/pages/tsdoc/doc_comment_syntax/) categorizing its level of maturity.
+
+- ["internal"](https://api-extractor.com/pages/tsdoc/tag_internal/) indicates an API that is intended strictly to be used inside the iTwin.js libraries - never by users.
+- ["alpha"](https://api-extractor.com/pages/tsdoc/tag_alpha/) indicates a highly-experimental API that is not yet ready for testing by users. These are rare in iTwin.js as we are typically interested in early feedback.
+- ["beta"](https://api-extractor.com/pages/tsdoc/tag_beta/) indicates an API currently under development. Users are encouraged to experiment with the API and provide feedback, but should not use it in production.
+- ["public"](https://api-extractor.com/pages/tsdoc/tag_public/) indicates a stable API suitable for use in production.
+
+In addition to any one of the above release tags, an API may also be tagged as ["deprecated"](https://api-extractor.com/pages/tsdoc/tag_deprecated/), indicating that it should no longer be used and may be removed in a future release. A deprecated tag is typically accompanied by information about the package version in which it became deprecated and guidance for adjusting existing usage of the API, e.g., what API to use instead. A "public" deprecated API will only be removed according to the [deprecation policy](#api-deprecation-policy).
+
+Only "public" and "beta" APIs are included in the [published documentation](https://www.itwinjs.org/reference/). An API typically starts out as "beta". It may evolve rapidly in response to feedback before stabilizing and being promoted to "public". On occasion, a "beta" API may be abandoned - not all experiments succeed.
+
+The "public API" of a package comprises the set of all APIs it contains that are marked with the "public" release tag. The package's public API enjoys stability guarantees provided by the [package versioning policy](#package-versioning-policy).
+
+## Package versioning policy
+
+The individual packages that constitute the iTwin.js libraries use [semantic versioning](https://semver.org/) to convey information about how their public APIs change over time.
```

Comment:
I would suggest adding "(symver)" after the symantic versioning link. The term is used later in the document, and without stating that "symantic versioning" is abbreviated "symver", to me that later use of "symver" is confusing. In other words, change this sentence to the following:

> The individual packages that constitute the iTwin.js libraries use [semantic versioning](https://semver.org/) (symver) to convey information about how their public APIs change over time.

Reply:
added.